SimulationCore/Simulation/AiAnalysis.cs

```diff
@@ -89,44 +89,10 @@ public static List<PlayerInfo> Analyse(byte[] file)
         /// <throws><see cref="RuleViolationException"/></throws>
         public static List<PlayerInfo> Analyse(byte[] file, bool checkRules)
         {
-            // setup appdomain
-            AppDomainSetup setup = new AppDomainSetup();
+            PlayerAnalysis analysisHost = new PlayerAnalysis();
```

Comment:
Moment mal. Bedeutet das, dass die Analyse nicht mehr in einer Appdomain passiert? Das würde bedeuten, dass alle analysierten KIs im Hauptprozessraum rumgammeln bis zum nächsten Neustart. Ist das so gewollt? Oder ist das dank Cecil nicht mehr notwendig?

Reply:
die Dlls werden eigentlich nicht geladen, da Cecil die direkt liest und nciht in die AppDomain lädt

Reply:
Top. Dann vergiss mal meinen Hinweis.

Reply:
die AppDomain ist also nur ncoh für das wirkliche ausführen wichtig